packages/endpoints-overlay/src/index.js

```diff
@@ -96,6 +111,7 @@ EndpointsOverlay.defaultProps = {
   forgetPlace: noop,
   rememberPlace: noop,
   locations: [],
+  MapMarkerIcon: undefined,
```

Comment:
Is it worth pulling the `DefaultMapMarkerIcon` out of the `endpoint.js` default props and into this file? Perhaps not since users of the library may want to use endpoint components independently from this overlay. But I think it's worth posing the question.

Reply:
Landon's idea sounds like a good one. But maybe better to ticket this for a future version. My preference at the moment is to get AltSource all the stuff they need (they demo'd their work yesterday, and this PR came up, due to the icons not matching the mockups).